OctoberCms.gitignore

```diff
@@ -0,0 +1,19 @@
+/bootstrap/compiled.php
+/vendor
+composer.phar
+.DS_Store
+.idea
```

Comment:
We use [`Global\JetBrains.gitignore`](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore) to track rules specific to the JetBrains family of IDEs. Please omit this rule from your template to prevent unnecessary duplication.